docker/glibc/build_scripts/remove-vsyscall.patch

```diff
@@ -0,0 +1,401 @@
+diff --git a/BUILD/glibc-2.12-2-gc4ccff1/nptl/sysdeps/unix/sysv/linux/x86_64/lowlevellock.S b/BUILD/glibc-2.12-2-gc4ccff1/nptl/sysdeps/unix/sysv/linux/x86_64/lowlevellock.S
```

Comment:
Who wrote this patch?

I found this commit... is it from here?  https://github.com/markrwilliams/manylinux/commit/e9493d55471d153089df3aafca8cfbcb50fa8093#diff-3eda4130bdba562657f3ec7c1b3f5720

I'm not sure who wrote it, if they need credit, or if it needs review.

Reply:
I guess @markrwilliams wrote it ? in #152

Reply:
I did, largely backporting vsyscall-less code from newer glibcs.